items.json

```diff
@@ -18196,9 +18259,30 @@
     },
     "weapons": []
   },
+  {
+    "id": "crpg_mount_charger_11",
+    "name": "Bucephalus",
+    "culture": "Khuzait",
+    "type": "Mount",
+    "price": 71264,
+    "weight": 450.0,
+    "tier": 11,
+    "requirement": 0,
+    "flags": [
+      "Civilian"
+    ],
+    "mount": {
+      "bodyLength": 105,
+      "chargeDamage": 8,
+      "maneuver": 72,
+      "speed": 44,

```

Comment:
you're not allowed to manually edit the tier. The tier is calculated through the exporter. You have to adapt the stats to reflect the tier

Reply:
Righto. Now you mention it, that tier calculation rings a bell. I would have assumed it would be a case of leaving the tier fields blank, putting items.json and horses, etc into module data, loading up the exporter, then seeing what is calculated on exporting? I'll have to ask someone in the discord about it, as I'm currently crashing due to "Line 20 Position 2 and Line 410 something".